scripts/download-metamath

```diff
@@ -36,9 +36,15 @@ cd ..
 # Store latest_version value inside metamath/ so we can get it later.
 printf '%s' "$latest_version" > metamath/.latest_version
 
-rm -f symbols.tar.bz2
-wget -N -q http://us.metamath.org/downloads/symbols.tar.bz2
+rm -f symbols.tar.bz2 symbols.tgz
+
+# Get symbols file. We used to get it from the Metamath website, but now the
+# website is generated from version controlled files, so we need to
+# get the symbols from our version controlled sources.
+# wget -N -q http://us.metamath.org/downloads/symbols.tar.bz2
+
+wget -N -q -o symbols.tgz https://github.com/metamath/symbols/tarball/main
```

Comment:
Why `-q` ? It can be informative to see the detailed output of the download. I do not know if there is a speed difference, but if there is, it is probably minor.

Reply:
Only because it was -q before. Happy to remove it, though hopefully it won't matter :-).